src/canvas.ts

```diff
@@ -0,0 +1,177 @@
+import utils from './utils';
+import Polygon from './polygon';
+
+import ISetting from './interfaces/canvas-setting.interface';
```

Comment:
Separate file for interface doesn't make much sense, you can keep it here instead